calc.py

```diff
@@ -0,0 +1,43 @@
+import math
+
+def add(a, b):
+    return a+b
+
+def subs(a, b):
+    return a-b
+
+def multiply(a, b):
+    return a*b
+
+def divide(a, b):
+    return a/b
+
+def root(a):
+    return math.sqrt(a)
+
+def square(a):
+    return a*a
+
+def sin(a):
+    return math.sin(a)
+
+def cos(a):
+    return math.cos(a)
+
+
+
+if __name__ == '__main__':
+    while True:
```

Comment:
The request was to read the whole line (add 10 20 30 40), parse it and call the right function (only one function)